sass/ohos/_tags.scss

```
@@ -12,28 +17,30 @@
         text-decoration: none;
     }
 
+    background-color: var(--tag-theme-color);
```

It might be good to have a fallback here, something like:

```suggestion
    background-color: var(--tag-theme-color, #ccc);
```

---

Set to use the same fallback colour as the wikidata overlays. All tags _should_ have a colour, but makes sense to be consistent